### ISSUE TITLE
Feature Request: Add option to display date

### ISSUE THREAD

**Issue by GlennSchner** [dremin/RetroBar#764]
What it says on the tin. An option to display the date alongside the time, in the number of usual formats, would be handy. Opening the clock instead of the notification center (never use this) would be nice too.

**Comment by 111ebobo** [comment]
> What it says on the tin. An option to display the date alongside the time, in the number of usual formats, would be handy. Opening the clock instead of the notification center (never use this) would be nice too.

Put your taskbar sideways

**Comment by SeventySev3n** [comment]
I agree, it would be super nice to have the option to do this! I got so used to it since Windows 7 lol
